### ISSUE TITLE
Support schedule for CenterFormer

### ISSUE THREAD

**Issue by QingXIA233** [open-mmlab/mmdetection3d#2306]
### Model/Dataset/Scheduler description

CenterFormer is a center-based transformer network for 3D object detection, which achieves state-of-the-art performance for a single model on the Waymo Open Dataset, with 73.7% mAPH on the validation set and 75.6% mAPH on the test set. It achieves better performances than CenterPoint in both single-frame and multi-frame settings.

### Open source status

- [X] The model implementation is available
- [ ] The model weights are available.

### Provide useful links for the implementation

The official implementation is based on Det3D framework in https://github.com/TuSimple/centerformer.

**Comment by JingweiZhang12** [comment]
The scheduler has been supported: https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/projects/CenterFormer/configs/centerformer_voxel01_second-attn_secfpn-attn_4xb4-cyclic-20e_waymoD5-3d-3class.py#L249-L289

**Comment by QingXIA233** [comment]
Wow, thank you! Really appreciate your works.

**Comment by Eaphan** [comment]
Hi, Jingwei. Thanks for the implementation of centerformer. When I use the config, I can train the model but when I run test.py, the model is not rightly evaluated. The checkpoint is loaded and no inference is done. Is the config complete? @JingweiZhang12. 
![image](https://user-images.githubusercontent.com/21019429/223001807-7c89e26d-0fc7-4047-b339-9a4aa2c8d7d7.png)
